go.mod

```diff
@@ -0,0 +1,3 @@
+module decred.org/dcrdex-assets
```

Comment:
@dajohi 

```
$  curl https://decred.org/dcrdex-assets?go-get=1
<meta name="go-import" content="decred.org/dcrdex git https://github.com/decred/dcrdex">
```

Can we revise this for the dcrdex-assets repo?  This needs adjusting, correct, @jrick?

Reply:
yes, it's matching incorrectly. will need some fixes on the webserver, but nothing i can do here on this pr about it.

Reply:
Understood, was just poking around